src/scss/components/_dynamic_list.scss

```diff
@@ -0,0 +1,56 @@
+.dynamic-list {
+  &__title {
+    font-size: 21px;
+    line-height: 28px;
+    font-weight: 400;
+  }
+
+  &__container {
+    overflow: auto;
+  }
+}
+
+.dynamic-list-item {
+  display: flex;
+  list-style: none;
+  padding: $magicnumber1;
+  border-top: 1px solid $grey4;
+  margin: auto;
+
+  &__icon {
+    vertical-align: middle;
+    width: 1.2em;
+    height: 1.2em;
+    margin: auto $magicnumber1 auto 0;
+    flex-shrink: 0;
+  }
+
+  &__title {
+    font-weight: 600;
+    margin-top: auto;
+    margin-bottom: auto;
+  }
+
+  &__desc {
+    margin-top: auto;
+    margin-bottom: auto;
+    margin-right: $magicnumber1;
+    overflow-wrap: anywhere;
+  }
+
+  &__btn {
+    float: right;
+    margin: auto 0 auto auto;
+    padding: $magicnumber1;
+    background-color: $grey5;
+
+    &:focus {
+      outline: 3px solid $focus;
+    }
+  }
+
+  &__separator {
+    margin: auto $magicnumber1 auto $magicnumber1;
```

Comment:
We can simplify this by having `margin: auto $magicnumber1` - this shorthand should set top and bottom margins to `auto`, and left and right margins to the value of `magicnumber1`